README.md

```diff
@@ -2,31 +2,162 @@
+        /* We want to print out a message to the user
+           so, we print out the message "Hello World" */
+        System.out.println("Hello World");
+    }
+}
+```
+
+Multi-line comments are also helpful if we want to possibly comment out code.
+We may comment out code to isolate a block of it to test or look for any
+errors.
+
+```java
+public class SimpleSample {
+    public static void main(String[] args) {
+        /* Commenting out this line so we can print a different message
+        System.out.println("Hello World");
+         */
+        
+        System.out.println("I hope you are well today!");
+    }
+}
+```
+
+## Documentation Comments
+
+**Documentation comments** are comments that embed information about the code
+into the program itself. This is a way for programmers to clearly communicate
+and document their code in a readable format. It uses a similar syntax to
+multi-line comments. Documentation comments start with a forward slash
+followed by two asterisks (`/**`) and are closed with an asterisk followed by
+a forward slash (`*/`). The text will lay between the `/**` and `*/` and each
+new line of text will start with a space and an asterisk (`*`).
+
+Documentation comments also make use of the javadoc utility. **javadocs** are
+tags or annotations that help extract the information and put it into an HTML
+file. Consider the table of javadoc tags below:
+
+![javadoc annotations](https://curriculum-content.s3.amazonaws.com/java-mod-1/comments/Javadoc-Annotations.png)
+
+[Reference: Java: A Beginner's Guide, 9th Edition](https://learning.oreilly.com/library/view/java-a-beginners/9781260463569/appb.xhtml#lev1_385)
+
+Annotations that start with the at-symbol (@) are called **block tags** or
+**stand-alone tags** and must be at the beginning of a documentation comment
+line. Annotations surrounded by curly-braces ({@code}) are **inline tags** to
+be used in a larger description. We will see some of these tags used more than
+others, but it is good to keep this chart handy in case we ever get our hands
+on the Oracle Java documentation - that is the documentation for the Java
+programming language. That documentation makes use of these annotations quite
+frequently.
+
+Let's look at our `SimpleSample` class again and apply some documentation
+comments.
+
+```java
+/**
+ * This class is a simple class that prints
+ * a sample message to the console.
+ * @author Flatiron School
+ * @version 1.0
+ */
+public class SimpleSample {
+    
+    /**
+     * Main method for the class to be made "runnable"
+     * @param args : String[] - array of arguments
+     */
+    public static void main(String[] args) {
+        System.out.println("Hello World");
+    }
+}
+```
+
+Note that we started the documentation comments with a main description of the
+class or method to follow. Documentation comments will more likely be attached
+to classes and methods rather than individual variables themselves.
 
 ## Resources
```

Comment:
Add a reference that isn't behind a paywall. Everything else looks great!
Feel free to merge once you add the reference.

Reply:
@AlveeM - Can you double check this reference (sign out from O'Reilly account prior). I just did a quick Google search of "javadoc tags" and this was the first image that popped up. 

[Java, A Beginner's Guide, 5th Edition](https://www.oreilly.com/library/view/java-a-beginners/9780071606325/appblev1sec1.html)